### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by NunoFilipeSantos**

### CODE REVIEW

**Review by ekpyron — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -22,11 +22,9 @@
 <section class="highlight-section-blue">
 	<div class="container flex">
 		<div class="text editable">
-			<h2><strong>Solidity v0.8.17 is here.</strong></h2>
-			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.17">Solidity 0.8.17</a> fixes an important bug, makes overflow checks on multiplication more efficient and adds an LSP feature to
-always analyze all files in a project.
-			<br><br>For all details please refer to the <a href="https://blog.soliditylang.org/2022/09/08/solidity-0.8.17-release-announcement/">release announcement</a>. For details on the bugs, see the
-				security alert post on <a href="https://blog.soliditylang.org/2022/09/08/storage-write-removal-before-conditional-termination/">Storage Write Removal Bug On Conditional Early Termination</a>.</p>
+			<h2><strong>Solidity v0.8.18 is here.</strong></h2>
+			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.18">Solidity 0.8.18</a> includes the following notable features: Disabling CBOR metadata, Support for Paris Hardfork, Deprecation of selfdestruct and Changes in EVM bytecode generation from Yul
```
Comment (ekpyron) [inline index.html:26]: I approved, but... not sure ``Disabling CBOR metadata`` is understandable :-). And capitalization is very weird here :-).

**Review by cameel — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (cameel) [reply index.html:26]: Maybe "A setting for disabling CBOR metadata"? It does sound as if we were unconditionally disabling it :)

Comment (NunoFilipeSantos) [inline index.html:26]:
```suggestion
			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.18">Solidity 0.8.18</a> includes the following notable features: disabling CBOR metadata, support for Paris hardfork, deprecation of selfdestruct and changes in EVM bytecode generation from Yul
```

File: index.html [hunk 2]
```diff
@@ -22,11 +22,9 @@
 <section class="highlight-section-blue">
 	<div class="container flex">
 		<div class="text editable">
-			<h2><strong>Solidity v0.8.17 is here.</strong></h2>
-			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.17">Solidity 0.8.17</a> fixes an important bug, makes overflow checks on multiplication more efficient and adds an LSP feature to
-always analyze all files in a project.
-			<br><br>For all details please refer to the <a href="https://blog.soliditylang.org/2022/09/08/solidity-0.8.17-release-announcement/">release announcement</a>. For details on the bugs, see the
-				security alert post on <a href="https://blog.soliditylang.org/2022/09/08/storage-write-removal-before-conditional-termination/">Storage Write Removal Bug On Conditional Early Termination</a>.</p>
+			<h2><strong>Solidity v0.8.18 is here.</strong></h2>
+			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.18">Solidity 0.8.18</a> includes the following notable features: disabling CBOR metadata, support for Paris hardfork, deprecation of selfdestruct and changes in EVM bytecode generation from Yul
```
Comment (NunoFilipeSantos) [inline index.html:26]:
```suggestion
			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.18">Solidity 0.8.18</a> includes the following notable features: a setting for disabling CBOR metadata, support for Paris hardfork, deprecation of selfdestruct and changes in EVM bytecode generation from Yul
```

File: index.html (hunk 1 above)
Comment (NunoFilipeSantos) [reply index.html]: Fixed.

**Pull request merged**